frontend/js/ci.js

```diff
@@ -13,6 +13,14 @@ const formatDateTime = (date) => {
     return '' + dateString + ' | ' + timeString
 }
 
+const formatDateTimeShort = (date) => {
+    var d = date.getDate();
+    var m = date.getMonth() + 1;
+    var y = date.getFullYear().toString().substr(-2);
+
+    return '' + addZed(d) + '.' + addZed(m) + '.' + y;
+}
+
```

Comment:
we have a very similar function in index.js. Please refactor this so that both use the same function. Which one I have no preference.
The date function should then live in the main.js